File: src/com/novell/android/yastroid/SystemMessage.java
package com.novell.android.yastroid;

public class SystemMessage {
	
	private String message;
	private String module;
	private String date;

	public SystemMessage( String message, String module, String date) {
		this.message = message;
		this.module = module;
		this.date = date;
	}
	
	public String getMessage() {
		return message;
	}

	public String getModule() {
		return module;
	}

	public String getDate() {
		return date;
	}

}

File: mono/YaSTroid/YaSTroid/SystemMessage.cs

namespace YaSTroid
{
	public class SystemMessage
	{	
		private string message;
		private string module;
		private string date;

		public SystemMessage( string message, string module, string date)
		{
			this.message = message;
			this.module = module;
			this.date = date;
		}
		
		public string getMessage()
		{
			return message;
		}

		public string getModule()
		{
			return module;
		}

		public string getDate()
		{
			return date;
		}
	}
}
